client/app/telecom/telephony/alias/configuration/agents/ovhPabx/telecom-telephony-alias-configuration-agents-ovhPabx.controller.js

```diff
@@ -168,15 +170,19 @@ angular.module('managerApp').controller('TelecomTelephonyAliasConfigurationAgent
       }
     }
 
+    getInternalNumber(newNumber) {
+      const allNumbers = _(this.groupList).map('services').flatten().value();
```

Comment:
```suggestion
      const allNumbers = _.flatten(
        _.map(
          this.groupList,
          'services',
        ),
      );
```

to help transition to Lodash 4 :)